### ISSUE TITLE
Chapter 8 Solutions

### ISSUE THREAD

**Issue by cjoanc27** [keithmcnulty/peopleanalytics-regression-book#25]
How do we interpret the random effects table when there's a random slope to be examined? 

An example is the table below when I attempted to explore the random effects of individuals on the slope efficient of `intel`. Does a small variance indicate little random slope effect? Also what does the column `Corr` refer to? Thanks.
```
Random effects:
 Groups Name        Variance Std.Dev. Corr 
 iid    (Intercept) 12.5301  3.5398        
        intel        0.1108  0.3328   -0.73
Number of obs: 3890, groups:  iid, 268
```
My solutions for other exercises as attached. [Chapter 8.pdf](https://github.com/keithmcnulty/peopleanalytics-regression-book/files/8800552/Chapter.8.pdf)

**Comment by keithmcnulty** [comment]
Hi Jo

The random slope variance gives you information for how the slope varies by individual around the mean (which is the fixed effect parameter).  So you can only interpret it relative to the fixed slope.  You'll notice from your result that for pretty much all of the slopes you tested, the variance is a certain proportion of the fixed slope (it looks like around 10-30%), which is a sign that the slopes are fairly stable across individuals.  This would suggest that there is not a lot of interpretative value to be gained from random slopes here.  People might come into the speed dating experiment at very different starting points in terms of their likelihood of being interested in someone, but once they are in the experiment, how they are affected by the various factors doesn't vary that much.

The `Corr` column is generally not of great concern unless you are an academic statistician.  In general, we should not treat the intercept and slope as independent variables in a mixed model.  In this case (two fixed effects variables), they should be taken from a bivariate normal distribution as opposed to two independent univariate normal distributions.  Therefore it is important to know the correlation between them in order to appropriately estimate the model.  In practice though, `Corr` is not a piece of information that needs to be interpreted.

Hope that helps!

**Comment by cjoanc27** [comment]
Hi Keith, 
Thanks for your response. To follow up on random slope, is it that we also look at AIC to determine whether adding the random slope would help improve the model on top of adding the random intercept? Or we just rely on the random slope variance to intuitively justify the model selection (with or without random slope)? 

Thanks,
Jo

**Comment by keithmcnulty** [comment]
AIC has a different role in this situation, because we are not really thinking about model parsimony in the traditional sense (ie asking whether or not we should include a variable in the model).  In this case, AIC will tell you whether or not the model with or without a random slope is a better fit, which is useful for predictive purposes, but not that helpful in terms of interpretation of the model in this case.  For example, you might find that the AIC is a little lower when you include a random slope, but if the random slope variance is not particularly high, then you can still conclude that it's not very valuable to include it in terms of providing more insight about the problem.

So in short I would look at the random slope variance to determine whether or not adding a random slope is useful from an interpretation perspective.

**Comment by cjoanc27** [comment]
Thank you - exactly to your point that I found AIC slightly dropped when I included a random slope and that's why the question. This is very helpful. I appreciate it.
